modules/peers.js

```diff
@@ -335,6 +335,8 @@ __private.insertSeeds = function(cb) {
 					return setImmediate(eachCb);
 				});
 			}
+
+			return setImmediate(eachCb);
```

Comment:
If peer is not banned then the `eachCb` will be called twice (once inside the status callback and also here).

Reply:
Yes, it is in progress guys. Patience...